src/EditorFeatures/Test2/Diagnostics/UseAutoProperty/UseAutoPropertyTests.vb

```diff
@@ -92,5 +94,54 @@ end class
 </text>.Value.Trim()}
                 })
         End Function
+
+        <Fact(), Trait(Traits.Feature, Traits.Features.CodeActionsUseAutoProperty)>
+        Public Async Function TestLinkedFile() As System.Threading.Tasks.Task
```

Comment:
Validated this test fails before my change (showing the 'linked file marge' comment that @AArnott saw).